lib/gears/math.lua

```diff
@@ -79,5 +79,16 @@ function gmath.round(x)
     return math.floor(x + 0.5)
 end
 
+--- Round a number to a 10s place
+-- @class function
+-- @name roundf
+-- @tparam number v
+-- @tparam[opt=1] number bracket 10s place to round to
```

Comment:
So.... this is supposed to be 1, 10, 100, 0.001(?), or what?
I am not sure, but I feel like 0, 1, 2, 3 is more common for this (meaning: x'd place after the comma)

Reply:
Yes, you can round to any place you want. Rounding to 1 is default. If we change to 0, 1, 2, 3, etc. then we add an exponent operation in (which isn't slow, but just an extra op we can avoid).